build.gradle

```diff
@@ -87,7 +85,7 @@ processResources {
     // this will ensure that this task is redone when the versions change.
     inputs.property 'version', project.version
     inputs.property 'mcversion', minecraft_version
-
+    duplicatesStrategy = 'include'
```

Comment:
Why was this added?

Reply:
I've added a comment explaining this on commit #d4006fb:
duplicate strategy is necessary, but i'm not sure why. gradle complains that accesstransformer.cfg is a duplicate...

I thought that's maybe because of the new modules system but I really don't know

Reply:
I've tested it again and now it works without duplicatesStrategy, therefore I removed it